internal/domain/file_content_response.go

```diff
@@ -0,0 +1,9 @@
+package domain
+
+type FileContentResponse struct {
+	Filename  string `json:"filename"`
+	Content   string `json:"content"`
```

Comment:
Consider adding validation for StartLine and EndLine. If EndLine is less than StartLine, it may lead to logical errors in the usage of this struct.